### PR TITLE
Nikita / FE changes for Phase1 (upd)

### PR DESCRIPTION
**Pull request by nikita-deriv**
Update home page for Deriv API with suggested [changes](https://redmine.deriv.cloud/issues/36828#Nikita_/_FE_changes_for_Phase1)

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -2050,6 +2007,11 @@ pre {
   color: #fafafa;
   background-color: #0e0e0e;
 }
+
+.gray{
```
Comment (mustofa-binary) [inline css/main.scss:2011]: nit: space

File: _pages/index.html [hunk 2]
```diff
@@ -68,72 +62,38 @@ <h3>Read our guide</h3>
...
-      />
-      <h2>Need code samples?</h2>
-      <p class="mb-0">
-        See <a href="{{ branch }}/docs/code-samples/">code samples</a> in
-        different languages.
-      </p>
-    </div>
+    <p class="term-conditions">By using our API, you confirm that you have read and agreed to our
+      <a href="https://deriv.com/tnc/business-partners-api-user.pdf" target="_blank" rel="noopener noreferrer">terms and
+        conditions.</a>
```
Comment (mustofa-binary) [inline _pages/index.html:71]: can <a in one line?

File: css/main.scss (hunk 1 above)
Comment (nikita-deriv) [reply css/main.scss:2010]: Removed lines between the styles.

File: _pages/index.html (hunk 2 above)
Comment (nikita-deriv) [reply _pages/index.html]: Yes, placed it in one.

**Pull request merged**